src/syncAzure.builds

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Yes src/publish.builds shouldn't be a .builds file either. They should just be .proj files because they are single project and are about building a set of projects which is what a .builds file is about.